index.html

```diff
@@ -156,11 +156,11 @@ <h4 class="display-5 section-title">What is the Synfig Studio 2D animation Hacka
          </div>
          <div class="mx-auto bd-highlight px-3">
             <p class="text-justify">During the Hackathon the participants will create an original 2D animated short movie of maximum of 5 mins duration using Synfig Studio. You should make the animation (graphics, audio, etc.) using ONLY open source software. You can make the narration in (----) Indian languages for your animation. 
-            </p>
-            <!-- <div class="p-2 pt-1 pb-1 text-center">
-               <figure><img src="image/Process_flow5.png" class="img-fluid" style="width:650px;" alt="process_flow">
+            </p><br><br>
+            <div class="p-2 pt-1 pb-1 text-center">
+               <figure><img src="image/flowchart.png" class="img-fluid" style="width:650px;" alt="process_flow">
```

Comment:
Move flowchart to "Eligibility to participate" section

Reply:
Done